File: Agile Development Practices/Code/Java/videoworld/src/main/java/com/thoughtworks/videorental/domain/Rental.java
package com.thoughtworks.videorental.domain;

import com.thoughtworks.datetime.Period;

public class Rental {
	private final Movie movie;
	private final Customer customer;
	private final Period period;

	public Rental(Customer customer, Movie movie, Period period) {
		this.movie = movie;
		this.customer = customer;
		this.period = period;
	}

	public Movie getMovie() {
		return movie;
	}

	public Customer getCustomer() {
		return customer;
	}

	public Period getPeriod() {
		return period;
	}
}

File: Agile Development Practices/Code/DotNet/videoworld/VideoWorld/Models/Rental.cs
using System;

namespace VideoWorld.Models
{
    public class Rental 
    {
        private readonly Movie movie;
        private readonly int periodInDays;

        public Rental(Movie movie, int periodInDays)
        {
            this.movie = movie;
            this.periodInDays = periodInDays;
        }

        public Movie Movie
        {
            get {
                return movie;
            }
        }

        public int Period
        {
            get { return periodInDays; }
        }
    }
}